docker/docker-compose.yml

```diff
@@ -201,7 +201,7 @@ services:
       <<: *base-core-service-env
```

Comment:
@anatoliynovo 

the service has to be defined in this compose. Just take another service (e.g. UM) as example and adjust the ENV's